### PR TITLE
Css frameworks

### PR DESCRIPTION
**Pull request by RiverMichael**

### CODE REVIEW

File: feed/index.html [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+    <link rel="icon" href="data:;base64,=" />
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-HwwvtgBNo3bZJJLYd8oVXjrBZt8cqVSpeBNS5n7C8IVInixGAoxmnlMuBnhbgrkm" crossorigin="anonymous"></script>
+    <script defer src="../js/script.js"></script>
+  </head>
+  <body class="bg-primary text-white min-vh-100 d-flex flex-column">
+    <header class="bg-primary sticky-top p-1">
+      <nav class="navbar navbar-expand-sm bg-primary" data-bs-theme="dark">
+        <div class="container-fluid">
+          <a href="#" class="navbar-brand">
```
Comment (Fermain) [inline feed/index.html]:
```suggestion
          <a href="/" class="navbar-brand">
```

This should link to your homepage.

File: src/scss/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
+// Variables
+
+// Gray-color-variables
+$white: #fffffffa !default;
```
Comment (Fermain) [inline src/scss/_variables.scss]: You can remove the `!default` flag from all of your custom variables.

**Review by Fermain — CHANGES_REQUESTED** [review]
This is an excellent submission with only a few minor changes required.

File: feed/index.html (hunk 1 above)
Comment (RiverMichael) [reply feed/index.html]: Even when the homepage is the login/signup page? I was thinking that the link then should go to the feed page as the homepage when you are in the application.

**Pull request merged**